src/abstractdataframe/sort.jl

```diff
@@ -357,9 +357,10 @@ end
 """
     sort(df::AbstractDataFrame, cols;
          alg::Union{Algorithm, Nothing}=nothing, lt=isless, by=identity,
-         rev::Bool=false, order::Ordering=Forward)
+         rev::Bool=false, order::Ordering=Forward, view::Bool=false)
 
-Return a copy of data frame `df` sorted by column(s) `cols`.
+Return a copy of data frame `df` sorted by column(s) `cols` unless
+`view=true` in which case a view into `df` is returned instead.
```

Comment:
```suggestion
Return a copy of data frame `df` sorted by column(s) `cols`.
If `view=true` a `SubDataFrame` view into `df` is returned instead.
```

We need to decide on a general policy to document `view`: here you added it to the first sentence, but for `filter` it's only mentioned later. It's tricky. Ideally `view` would only be mentioned later as it's not the core of what a function does, but since the first sentence often says "copy" it's hard to avoid. Maybe we should avoid "copy" in the first sentence in general (e.g. say "Sort data frame `df` by columns `cols`"; this is what the docstring for vectors says) so that `view` can be mentioned later?